### ISSUE TITLE
Aliased folders won't show

### ISSUE THREAD

**Issue by badda71** [lrsjng/h5ai#688]
This is the same issue as Nr 185. However, the suggested fix would be to give the user the possibility to define a mapping in the h5ai config to manually map aliased directories to the real directories. Would this be possible?

**Comment by shirshak55** [comment]
it works on nginx
which server are u using?

**Comment by badda71** [comment]
I'm using nginx. What exactly is working? Aliased folders are taken into account? If yes, how?

**Comment by shirshak55** [comment]
i just used ln -Sf command it was working . Are u using ubuntu or other server?

**Comment by shirshak55** [comment]
nginx aliasing don't work but aliased folder should have worked tho.

**Comment by badda71** [comment]
I'm not talking about filesystem links (with ln, you create links) - I'm talking about aliased foldes as configured in the server config. The solution for that would be to include a mapping in h5ai config where the admin can map aliased directory names to real directories.

**Comment by shirshak55** [comment]
@badda71 oh then its clearly written nginx aliasing don't work due to issue of something like ../../ path etc are hard to map etc.

**Comment by shirshak55** [comment]
To say what Irsjung said:

```
Does not work with aliased folders in general (as available in Apache httpd). Aliased folders make it impossible to map URLs to file system fold
```

**Comment by badda71** [comment]
That's what I'm talking about. I was suggesting a solution of how to get aliased folders working. Actually that was more a feature request than an issue :-)

**Comment by shirshak55** [comment]
I don't think Irsjng will add this feature as this repo has not seen major update :)

I think we need to do this ourself :) Do u know root cause of this problem?

**Comment by badda71** [comment]
Yeah, I created this "issue" before I realized that this project it not maintained anymore.
It's not a problem either, it's a feature request. I already checked - to implement it needs some adaptations in _h5ai/private/php/core/class-context.php, namely in functions to_href(), to_path() and maybe more.
I did quite a some enhancements in my fork of _h5ai but I have not yet gotten down to implementing this feature yet because I found a workaround for myself ;-)
